services/blockchain-connector/shared/sdk/schema.js

```diff
@@ -1,59 +1,64 @@
 /*
- * LiskHQ/lisk-service
- * Copyright © 2022 Lisk Foundation
- *
- * See the LICENSE file at the top-level directory of this distribution
- * for licensing information.
- *
- * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
- * no part of this software, including this file, may be copied, modified,
- * propagated, or distributed except according to the terms contained in the
- * LICENSE file.
- *
- * Removal or modification of this copyright notice is prohibited.
- *
- */
-const { getSchema } = require('./actions_1');
-
-const getAccountSchema = async () => {
-	const allSchemas = await getSchema();
-	return allSchemas.account;
-};
+* LiskHQ/lisk-service
+* Copyright © 2022 Lisk Foundation
+*
+* See the LICENSE file at the top-level directory of this distribution
+* for licensing information.
+*
+* Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+* no part of this software, including this file, may be copied, modified,
+* propagated, or distributed except according to the terms contained in the
+* LICENSE file.
+*
+* Removal or modification of this copyright notice is prohibited.
+*
+*/
+let schemas;
 
-const getBlockSchema = async () => {
-	const allSchemas = await getSchema();
-	return allSchemas.block;
-};
+const setSchemas = (_schemas) => schemas = _schemas;
 
-const getBlockHeaderSchema = async () => {
-	const allSchemas = await getSchema();
-	return allSchemas.blockHeader;
-};
+const getAccountSchema = async () => schemas.account;
 
-const getBlockHeaderAssetSchema = async (blockHeaderVersion) => {
-	const allSchemas = await getSchema();
-	const { blockHeadersAssets } = allSchemas;
-	return blockHeadersAssets[blockHeaderVersion];
-};
+const getBlockSchema = async () => schemas.block;
```

Comment:
Please consider making the getters synchronous as there are no async operations happening here. Please make sure to make the changes where these methods are getting called as well.